pkg/injector/sidecar/pod_patch_test.go

```diff
@@ -14,11 +14,14 @@ limitations under the License.
 package sidecar
 
 import (
+	"encoding/json"
 	"strconv"
 	"testing"
 
+	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
Please use `"github.com/evanphx/json-patch/v5"` (just realized there's the new version by looking at our go.mod)

Reply:
done